src/main/java/com/atlassian/db/replica/internal/ReplicaStatement.java

```diff
@@ -648,7 +657,8 @@ public ReplicaStatement build() {
                 resultSetType,
                 resultSetConcurrency,
                 resultSetHoldability,
-                readOnlyFunctions
+                readOnlyFunctions,
+                compatibleWithPreviousVersion
```

Comment:
I've found myself scrolling fast through the boiler code related to:`compatibleWithPreviousVersion`
It might be good idea (for review purposes) to extract separate commit - adding `compatibleWithPreviousVersion`